src/index.css

```
@@ -120,12 +163,97 @@
+ * reset-password "cards" rendered with no background, no border and no shadow:
+ * bare text floating on the page over the blurred blue bokeh those pages set
+ * up behind them. Somebody clearly intended a frosted panel, so define it
+ * rather than delete the intent.
+ *
+ * These live in `components`, not `utilities`, so existing per-element
+ * overrides (SignUp's `dark:bg-gray-900` on the back button, for example)
+ * still win — the utilities layer is emitted after this one.
+ */
+@layer components {
+  .glass-morphism {
+    background-color: rgb(255 255 255 / 0.72);
```

Stylelint (17.14.0)

[error] Expected "0.72" to be "72%" (alpha-value-notation)

(alpha-value-notation)